hardware_interface/test/test_robot_hardware_interface.cpp

```diff
@@ -120,12 +120,12 @@ TEST_F(TestRobotHardwareInterface, cannot_double_register_handles)
 TEST_F(TestRobotHardwareInterface, can_get_registered_joint_names)
 {
   SetUpHandles();
-  EXPECT_THAT(robot_.get_registered_joint_names(), ElementsAre(JOINT_NAME));
+  // EXPECT_THAT(robot_.get_registered_joint_names(), ElementsAre(JOINT_NAME));
```

Comment:
What's the status of these tests?

Reply:
They only test the `OperationModeHandle`. The joint handles tests are now in `test_joint_handle.cpp` and `test_register_joints.cpp`.